sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/NettyAsyncHttpClient.java

```diff
@@ -190,7 +195,24 @@ public HttpHeaders getHeaders() {
                 if (!reactorNettyConnection.isDisposed()) {
                     reactorNettyConnection.channel().eventLoop().execute(reactorNettyConnection::dispose);
                 }
-            }).map(ByteBuf::nioBuffer);
+            }).map(byteBuf -> {
```

Comment:
Should this be in the bodyIntern method instead?

Reply:
As discussed, changing bodyIntern will result in double copy for other `getBody*` methods that don't return `Flux<ByteBuffer>`